src/renamer/MovieRenamer.cpp

```diff
@@ -213,24 +214,97 @@ MovieRenamer::RenameError MovieRenamer::renameMovie(Movie& movie)
                     }
                 }
 
-                /*
-                QStringList filters;
-                for (const QString &extra: m_extraFiles)
-                    filters << baseName + extra;
-                for (const QString &subFileName: currentDir.entryList(filters, QDir::Files |
-                QDir::NoDotAndDotDot)) { QString subSuffix = subFileName.mid(baseName.length()); QString newBaseName
-                = newFileName.left(newFileName.lastIndexOf(".")); QString newSubName = newBaseName + subSuffix;
-                    ui->results->append(tr("<b>Rename File</b> \"%1\" to \"%2\"").arg(subFileName).arg(newSubName));
-                    if (!dryRun) {
-                        if (!rename(currentDir.canonicalPath() + "/" + subFileName, currentDir.canonicalPath() + "/"
-                + newSubName)) ui->results->append("&nbsp;&nbsp;<span style=\"color:#ff0000;\"><b>" + tr("Failed") +
-                "</b></span>"); else FilmFiles.append(newSubName);
+                // Handle all external subtitle files using preconfigured patterns
+                const mediaelch::FileFilter& subtitleFilters = Settings::instance()->advanced()->subtitleFilters();
```

Comment:
hmmm...

I do wonder why we need to look for new subtitles here. Shouldn't they already be part of `movie.subtitles()`? After all, you've adapted `src/file_search/movie/MovieDirectorySearcher.cpp`.

Reply:
it doesn't seem to find them as "movie.subtitles()" and so I think my new approach just deals with them as an aside. when I dump the vector right before line 218 `movie.subtitles()` shows empty. wdyt? I care less about the proper tracking of the movie assets than I do the renamer "just working"